lib/tasks/events_registrations.rake

```diff
@@ -1,4 +1,22 @@
 namespace :events_registrations do
+  desc 'Sets max_attendees to 1 or number of registrations, if require_registration is set'
+  task set_max_attendees: :environment do
+    @events = Event.all.where(require_registration: true, max_attendees: nil)
+    puts "Fixing max_attendees attribute for #{@events.length} events."
+    puts "The IDs of those events are: #{@events.pluck(:id)}"
+
+    @events.each do |event|
+      if event.registrations.any?
+        event.max_attendees = event.registrations.count
+      else
+        event.max_attendees = 1
```

Comment:
Not really, but I am not sure what the alternative is.
We have no way of knowing the value of max_attendees. OK it is probably not going to be 1, but we don't know if it is 10 or 30 or...